kubernetes/victimplay.sh

```diff
@@ -1,10 +1,14 @@
-#!/bin/sh -e
+#!/bin/bash -e
```

Comment:
Do we want to error on unset variables?
```suggestion
#!/bin/bash -eu
```